src/Perlang.Common/Compiler/CompilerFlags.cs

```diff
@@ -15,5 +15,11 @@ public enum CompilerFlags
     /// This flag disables all caching of compiled Perlang code, to ensure that all parts of the compilation is being
     /// used.
     /// </summary>
-    CacheDisabled = 1
+    CacheDisabled = 1,
+
+    /// <summary>
+    /// This flag which makes the compiler remove the `main` method if it is empty. This
+    /// can be useful e.g. when the `main` method is implemented in C++.
+    /// </summary>
+    RemoveEmptyMainMethod = 2
```

Comment:
This was necessary to be able to write a Perlang program which is "bootstrapped" by a C++ `main` method implementation. :slightly_smiling_face: It's not an incredibly important feature, but it was useful to get a simple test in place without all the mechanisms for interfacing between Perlang and C++ code in place.